package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "panda-9000",
-  "version": "2.0.0-alpha-02",
+  "version": "2.0.0-alpha-03",
```

Comment:
The version shouldn't be bumped until you are ready to run NPM publish. Someone else's PR could be merged before you and they would get `alpha-03` and yours would then become `alpha-04`.